catalystwan/models/monitoring/security_policy.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2024 Cisco Systems, Inc. and its affiliates
+
+from pydantic import BaseModel, Field
+
+
+class DeviceListResponse(BaseModel):
```

Comment:
why not to model this in regular way?
![image](https://github.com/cisco-open/cisco-catalyst-wan-sdk/assets/116343782/92cd2af6-e79a-46ff-9eda-ce635d9e94ab)
```suggestion
class SecurityPolicyDeviceList(BaseModel):
    amp_down: List[str]
    amp_up: List[str]
    ips_down: List[str]
    ips_up: List[str]
    urlf_down: List[str]
    urlf_up: List[str]
    zbfw_down: List[str]
    zbfw_up: List[str]
```

Reply:
Have made these changes